src/Hl7.Fhir.R5.Core/Specification/Validation/PocoValidationExtensions.cs

```diff
@@ -24,22 +21,23 @@ public static class PocoValidationExtensions
     {
         public static OperationOutcome Validate(this Validator me, Base instance)
         {
-            return me.Validate(instance.ToTypedElement());
+            return me.Validate(instance.ToTypedElement(), ModelInfo.ModelInspector);
```

Comment:
If we'd have an overload for `Validator.Validate(Base b)` (I am pretty sure we have it), we would not need this overload with the ModelInspector?  Because then the validator can derive the modelinspector from the instance passed in with the new `ModelInspector.ForInstance()`?

Reply:
basically, we would "push this code down to the validator" instead of repeating the `ModelInfo.ModelInspector` here everywhere?

Reply:
We decided to remove the `ModelInspector.ForInstance()`.